resolwe/flow/views/entity.py

```diff
@@ -156,3 +162,23 @@ def patched_get_queryset():
         resp = super().update(request, *args, **kwargs)
         self.get_queryset = orig_get_queryset
         return resp
+
+    @list_route(methods=['post'])
+    def duplicate(self, request, *args, **kwargs):
+        """Duplicate (make copy of) ``Entity`` models."""
+        inherit_collection_membership = request.data.get('inherit_collection_membership', False)
```

Comment:
Why not just `inherit_collections`?

Reply:
Because it's not explicit enough. Current form rings a bell sooner.

Reply:
`inherit_collection_membership ` is too long for my taste, try to find something shorter, e.g. `inherit_collections`, `keep_collections`,...